packages/manager/apps/pci-kubernetes/src/pages/list/useDatagridColumn.tsx

```diff
@@ -0,0 +1,72 @@
+import { DatagridColumn, DataGridTextCell } from '@ovhcloud/manager-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsClipboard, OsdsLink } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useHref } from 'react-router-dom';
+import ActionsComponent from '@/components/listing/actions.component';
+import { TKube } from '@/types';
+
+export const useDatagridColumn = () => {
+  const { t } = useTranslation('listing');
+
+  const columns: DatagridColumn<TKube>[] = [
+    {
+      id: 'name',
+      cell: (props: TKube) => (
+        <DataGridTextCell>
+          <OsdsLink
+            color={ODS_THEME_COLOR_INTENT.primary}
+            href={useHref(`${props.id}`)}
+          >
+            {props.name}
+          </OsdsLink>
+        </DataGridTextCell>
+      ),
+      label: t('kube_list_name'),
+    },
+    {
+      id: 'id',
+      cell: (props: TKube) => (
+        <DataGridTextCell>
+          <OsdsClipboard
```

Comment:
no tooltip "copied" is shown on click

Reply:
Replaced it by the one coming from manger-component